.yamato/config/universal.metafile

```diff
@@ -1,59 +1,94 @@
 project:
   name: Universal
   folder: UniversalGraphicsTest
-test_platforms:
-  - type: Standalone
-    utr_flags:
-      - [OSX_Metal]: --timeout=2400
-    utr_flags_build:
-      #- [Android_OpenGles3, Android_Vulkan]: --timeout=2700
-      #- [Win_DX11, Win_DX12, Win_Vulkan]: --timeout=2000
-      #- [iPhone_Metal]: --timeout=1800
-      - [all]: --timeout=3000
-  - type: playmode
-  - type: playmode
-    name: playmode_cache
-    utr_flags:
+
+reuse_blocks:
+  setup_with_utr:
+    - [all]: <shared.platform.commands.unity_downloader_cli>
+    - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+    - [Win_DX11, Win_DX12, Win_Vulkan]: cd Tools && powershell -command ". .\Unity.ps1; Set-ScreenResolution -width 1920 -Height 1080"
+    - [all]: <shared.platform.commands.setup_utr>
+    - [all]: <shared.platform.commands.call_utr>
+  setup_without_utr:
+    - [all]: <shared.platform.commands.unity_downloader_cli>
+    - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+    - [Win_DX11, Win_DX12, Win_Vulkan]: cd Tools && powershell -command ". .\Unity.ps1; Set-ScreenResolution -width 1920 -Height 1080"
+    - [all]: <shared.platform.commands.setup_utr>
+  cache_server:
```

Comment:
could these cache server commands maybe move to the shared metafile? not a big deal either way

Reply:
they can, it'd be nice indeed, but I'm not sure how quick it'd be and if it would hold back the PR or not. Maybe we can do this later as a separate PR?

Reply:
yeah thats fine